### PR TITLE
#864 DataStore#exist() should return false if id not exist

### PR DESCRIPTION
**Pull request by vitojeng**
required by #864

### Checklist
- [ ] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [ ] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [ ] Add Assignees (of course, you are the assignee by default!)
- [ ] DON'T set a Milestone
- [ ] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [ ] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
@vitojeng Could you add a test ?

**Comment by vitojeng** [comment]
> @vitojeng Could you add a test ?

Sure.

File: ohara-configurator/src/test/scala/com/island/ohara/configurator/store/TestDataStore.scala [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+private[store] final case class TestData(id: String, name: String, lastModified: Long, kind: String) extends Data
+
+class TestDataStore extends MediumTest with Matchers {
+
+  private[configurator] val DATA_SERIALIZER: Serializer[Data] = new Serializer[Data] {
```
Comment (chia7712) [inline ohara-configurator/src/test/scala/com/island/ohara/configurator/store/TestDataStore.scala]: useless object?

File: ohara-configurator/src/test/scala/com/island/ohara/configurator/store/TestDataStore.scala [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+import com.island.ohara.common.util.{CommonUtils, Releasable}
+import com.island.ohara.configurator.Configurator
+import org.junit.{After, Test}
+import org.scalatest.Matchers
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+private[store] final case class TestData(id: String, name: String, lastModified: Long, kind: String) extends Data
```
Comment (chia7712) [inline ohara-configurator/src/test/scala/com/island/ohara/configurator/store/TestDataStore.scala]: Don't use "Test" as prefix for the class which doesn't carry test case.

Comment (vitojeng) [reply ohara-configurator/src/test/scala/com/island/ohara/configurator/store/TestDataStore.scala]: Got it!

**Pull request merged**